rmc-docs/src/tutorial/rmc-first-steps/src/lib.rs

```diff
@@ -46,15 +46,15 @@ mod tests {
     // ANCHOR_END: proptest
 }
 
-fn __nondet() -> u32 {
+fn rmc::nondet() -> u32 {
```

Comment:
You probably meant to remove this function.

Reply:
yep... thanks for catching it.